test/unit/query-generator/describe-table-query.test.ts

```diff
@@ -0,0 +1,420 @@
+      mssql: 'SELECT '
+        + `c.COLUMN_NAME AS 'Name', `
+        + `c.DATA_TYPE AS 'Type', `
+        + `c.CHARACTER_MAXIMUM_LENGTH AS 'Length', `
+        + `c.IS_NULLABLE as 'IsNull', `
+        + `COLUMN_DEFAULT AS 'Default', `
+        + `pk.CONSTRAINT_TYPE AS 'Constraint', `
+        + `COLUMNPROPERTY(OBJECT_ID(c.TABLE_SCHEMA+'.'+c.TABLE_NAME), c.COLUMN_NAME, 'IsIdentity') as 'IsIdentity', `
+        + `CAST(prop.value AS NVARCHAR) AS 'Comment' `
+        + 'FROM '
+        + 'INFORMATION_SCHEMA.TABLES t '
+        + 'INNER JOIN '
+        + 'INFORMATION_SCHEMA.COLUMNS c ON t.TABLE_NAME = c.TABLE_NAME AND t.TABLE_SCHEMA = c.TABLE_SCHEMA '
+        + 'LEFT JOIN (SELECT tc.table_schema, tc.table_name, '
+        + 'cu.column_name, tc.CONSTRAINT_TYPE '
+        + 'FROM INFORMATION_SCHEMA.TABLE_CONSTRAINTS tc '
+        + 'JOIN INFORMATION_SCHEMA.KEY_COLUMN_USAGE  cu '
+        + 'ON tc.table_schema=cu.table_schema and tc.table_name=cu.table_name '
+        + 'and tc.constraint_name=cu.constraint_name '
+        + `and tc.CONSTRAINT_TYPE='PRIMARY KEY') pk `
+        + 'ON pk.table_schema=c.table_schema '
+        + 'AND pk.table_name=c.table_name '
+        + 'AND pk.column_name=c.column_name '
+        + 'INNER JOIN sys.columns AS sc '
+        + `ON sc.object_id = object_id(t.table_schema + '.' + t.table_name) AND sc.name = c.column_name `
+        + 'LEFT JOIN sys.extended_properties prop ON prop.major_id = sc.object_id '
+        + 'AND prop.minor_id = sc.column_id '
+        + `AND prop.name = 'MS_Description' `
+        + `WHERE t.TABLE_NAME = 'myTable'AND t.TABLE_SCHEMA ='mySchema'`,
+      sqlite: 'PRAGMA TABLE_INFO(`mySchema.myTable`);',
+      db2: 'SELECT NAME AS "Name", TBNAME AS "Table", TBCREATOR AS "Schema", '
+        + 'TRIM(COLTYPE) AS "Type", LENGTH AS "Length", SCALE AS "Scale", '
+        + 'NULLS AS "IsNull", DEFAULT AS "Default", COLNO AS "Colno", '
+        + 'IDENTITY AS "IsIdentity", KEYSEQ AS "KeySeq", REMARKS AS "Comment" '
+        + 'FROM SYSIBM.SYSCOLUMNS '
+        + `WHERE TBNAME = 'myTable' AND TBCREATOR ='mySchema';`,
+      ibmi: `SELECT
+        QSYS2.SYSCOLUMNS.*,
+        QSYS2.SYSCST.CONSTRAINT_NAME,
+        QSYS2.SYSCST.CONSTRAINT_TYPE
+        FROM
+        QSYS2.SYSCOLUMNS
+        LEFT OUTER JOIN
+          QSYS2.SYSCSTCOL
+        ON
+          QSYS2.SYSCOLUMNS.TABLE_SCHEMA = QSYS2.SYSCSTCOL.TABLE_SCHEMA
+          AND
+          QSYS2.SYSCOLUMNS.TABLE_NAME = QSYS2.SYSCSTCOL.TABLE_NAME
+          AND
+          QSYS2.SYSCOLUMNS.COLUMN_NAME = QSYS2.SYSCSTCOL.COLUMN_NAME
+        LEFT JOIN
+          QSYS2.SYSCST
+        ON
+          QSYS2.SYSCSTCOL.CONSTRAINT_NAME = QSYS2.SYSCST.CONSTRAINT_NAME
+        WHERE QSYS2.SYSCOLUMNS.TABLE_SCHEMA = 'mySchema' AND QSYS2.SYSCOLUMNS.TABLE_NAME = 'myTable'`,
+    });
+  });
+
+  it('produces a DESCRIBE TABLE query with schema and schemaDelimiter argument', () => {
+    expectsql(() => queryGenerator.describeTableQuery('myTable', 'mySchema', '.'), {
```

Comment:
may want to use a different schema delimiter since `'.'` is already the default

This test is only relevant in dialects that don't support schemas

Reply:
Yeah, I tried that and came across the fact that only sqlite does something with this

Reply:
Yup, it's the only one that doesn't support schemas

Honestly I think we should, at some point, drop this fake schema support.